### PR TITLE
Fix Rejected applicant is displayed as Pending

### PR DESCRIPTION
**Pull request by gyroflaw**
Fix https://github.com/Joystream/pioneer/issues/3856

### CODE REVIEW

File: packages/ui/src/working-groups/components/ApplicationsList.tsx [hunk 1]
```diff
@@ -57,8 +57,10 @@ const ApplicationListItem = ({ application, past }: { application: WorkingGroupA
         return 'Hired'
       case 'ApplicationStatusWithdrawn':
         return 'Withdrawn'
```
Comment (traumschule) [inline packages/ui/src/working-groups/components/ApplicationsList.tsx:59]: This still hides "Withdrawn" (not hired) applicants: #3856

**Review by traumschule — CHANGES_REQUESTED** [review]

**Comment by ivanturlakov** [comment]
Tested on https://dao-git-fork-gyroflaw-fix-application-status-joystream.vercel.app/#/working-groups/my-applications
wss://34.230.5.182.nip.io/ws-rpc

Снимок экрана 2023-01-12 в 13 50 43

### ✅ ivant-tester-ca
Снимок экрана 2023-01-12 в 13 51 09
Снимок экрана 2023-01-12 в 13 51 24

### ✅ ivant-tester-c
Снимок экрана 2023-01-12 в 13 53 00
Снимок экрана 2023-01-12 в 13 53 08

@gyroflaw looks good 👍

**Review by traumschule — APPROVED** [review]
I hesitate to just approve this but likely a result of the runtime and out of scope.

In #4035 (storage-2) and [storage-3](https://dao-git-fork-gyroflaw-fix-application-status-joystream.vercel.app/#/working-groups/openings/storage-2) the application got automatically withdrawn when others were hired and as a result on the closed opening it now looks like `joystreamstats` never applied.

applied

withdrawn

**Review by thesan — APPROVED** [review]
LGTM!

**Pull request merged**